src/app/pages/send-tokens/send-tokens.tsx

```diff
@@ -26,12 +27,13 @@ import {
 import { logger } from '@shared/logger';
 import { Estimations } from '@shared/models/fees-types';
 import { RouteUrls } from '@shared/route-urls';
+import { useTransferableAssets } from '@app/store/assets/asset.hooks';
 
 import { SendTokensSoftwareConfirmDrawer } from './components/send-tokens-confirm-drawer/send-tokens-confirm-drawer';
 import { SendFormInner } from './components/send-form-inner';
-import { useTransferableAssets } from '@app/store/assets/asset.hooks';
 
 function SendTokensFormBase() {
+  useNextTxNonce();
```

Comment:
This is a side effect. Shouldn't we call a method, to get all relevant info, at the time at which we need it (generating the tx/displaying it to the user), rather than at the render time of this component?